taxi/urls.py

```diff
@@ -18,9 +24,33 @@
     ),
     path("cars/", CarListView.as_view(), name="car-list"),
     path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
+    path("cars/create/", CarCreateView.as_view(), name="car-create"),
+    path(
+        "cars/<int:pk>/update/", CarUpdateView.as_view(), name="car-update"
+    ),
+    path(
+        "cars/<int:pk>/delete/", CarDeleteView.as_view(), name="car-delete"
+    ),
+    path(
+        "manufacturers/create/",
+        ManufacturerCreateView.as_view(),
+        name="manufacturer-create",
+    ),
+    path(
+        "manufacturers/<int:pk>/update/",
+        ManufacturerUpdateView.as_view(),
+        name="manufacturer-update",
+    ),
+    path(
+        "manufacturers/<int:pk>/delete/",
```

Comment:
Consider adding a trailing slash to the 'manufacturers/<int:pk>/delete/' URL pattern to maintain consistency with Django's convention.